rhoc/pkg/cmd/connectors/connectors.go

```diff
@@ -22,7 +23,9 @@ func NewConnectorsCommand(f *factory.Factory) *cobra.Command {
 		list.NewListCommand(f),
 		delete.NewDeletesCommand(f),
 		get.NewGetCommand(f),
-		types.NeConnectorTypesCommand(f))
+		types.NeConnectorTypesCommand(f),
+		patch.NewReadyCommand(f),
```

Comment:
Maybe it should be `NewStartCommand` instead of `NewReadyCommand` considering the command line sub-command.

Reply:
@dhirajsb I see this is a consideration and I will implement it, but do you think a name change is a reason strong enough to block a PR?